packages/ember-glimmer/lib/environment.js

```diff
@@ -224,12 +227,15 @@ export default class Environment extends GlimmerEnvironment {
     return false;
   }
 
-  getComponentDefinition(path) {
+  getComponentDefinition(path, parentMeta = {}) {
```

Comment:
Lets not default it here, lets either ensure it is set once on the template and always there or check presence